packages/xchain-ethereum/src/client.ts

```diff
@@ -187,11 +178,14 @@ export default class Client implements XChainClient, EthereumClient {
    * @throws {"Phrase must be provided"}
    * Thrown if phrase has not been set before. A phrase is needed to create a wallet and to derive an address from it.
    */
-  getAddress = (): Address => {
-    if (!this.address) {
+  getAddress = (index = 0): Address => {
+    if (index < 0) {
+      throw new Error('index must be greater than zero')
+    }
+    if (!this.phrase) {
       throw new Error('Phrase must be provided')
     }
-    return this.address
+    return this.getWallet(index).address.toLowerCase()
```

Comment:
Any particular reason of removing the checksum casing of the address?

cf. https://coincodex.com/article/2078/ethereum-address-checksum-explained/